feature/mkt-imob-4: Recriando as tabelas do banco, refatorando a doc, modificando os servicos (#3)
erick-luiz/mkt-imoveis: README.md

* Preencha os dados: 
  * **Name**: Main (ou outro a seu critério)
  * **JDK** or JRE: Java 17 ou superior 
  * **Module**: imobiliaria.imobiliaria.main
  * **Main Class**: com.mkt.imobiliaria.ImobiliariaApplication
  * **Variaveis de Ambiente**: 
    * DB_URL=jdbc:postgresql://localhost:5432/imobiliaria;
    * POSTGRES_USER=postgres;
    * POSTGRES_PASSWORD=postgres

# 🔧 Code Style (Configure sua IDE)

> * Configurando o numero de espaços do TAB

Reviewer: ```suggestion
* Configurando o número de espaços do TAB
```

  * File > settings > Editor > Code Style > Java > Tabs and Idents
    * Tab size: 4
    * Ident: 4
    * Continuation ident: 8

* Configurando os imports
  * File > settings > Editor > Code Style > Java > Imports
    * Class count to use import with '*': 999 (Isso evita uso de *)
    * Name count to use static import with '*': 999

> * Configurando os imports automaticos e a reordenação

Reviewer: ```suggestion
* Configurando os imports automáticos e a reordenação
```

  * File > settings > Editor > General > Auto Imports
    * Marque a opção: Optimize imports on the fly 

# Teste Local 
 

> Após inicializar o serviço voce pode testar as interações com imoveis utilizando a coleção do postmam que está

Reviewer: ```suggestion
Após inicializar o serviço você pode testar as interações com imóveis utilizando a coleção do postman que está
```

dentro da pasta /doc. 

> Como para cadastrar um imovel é necessário ter uma imobiliaria cadastrada, use o sql abaixo para ter a 

Reviewer: ```suggestion
Como para cadastrar um imóvel é necessário ter uma imobiliária cadastrada, use o sql abaixo para ter a 
```

> imobiliaria e seu respectivo endereco cadastrados: 

Reviewer: ```suggestion
imobiliária e seu respectivo endereço cadastrados: 
```

> _Criando o endereço _

Reviewer: ```suggestion
_Criando o endereço_
```

```sql
INSERT INTO endereco (endereco_id, logradouro, numero, complemento, bairro, cidade, estado, cep, data_de_atualizacao, data_de_criacao)
VALUES
(nextval('endereco_seq'), 'Rua Teste', '100', 'Apto 101', 'Centro', 'São Paulo', 'SP', '01000-000', NOW(), NOW());
```

> _Com o id do endereço criado, crie a imobiliaria_

Reviewer: ```suggestion
_Com o id do endereço criado, crie a imobiliária_
```
